runtime/src/identity.rs

```diff
@@ -268,7 +263,7 @@ decl_module! {
 
             <DidRecords>::mutate( id,
             |record| {
-                (*record).signing_identities.retain( |s_id| ids_to_remove.iter().find( |id| s_id == *id).is_none());
+                (*record).remove_signing_identities( &ids_to_remove);
```

Comment:
Currently I think adding / removing signing identities doesn't trigger any events. We should have similar events (or reuse the same event, with a keytype discriminator) that we use for external signing keys.

Reply:
Fixed.